sunbird_service_api_test/src/test/java/org/sunbird/integration/test/malformedRequest/MalformedRequestTest.java

```diff
@@ -0,0 +1,121 @@
+package org.sunbird.integration.test.malformedRequest;
+
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.http.HttpStatus;
+import org.sunbird.common.util.Constant;
+import org.sunbird.integration.test.user.EndpointConfig.TestGlobalProperty;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.context.TestContext;
+import com.consol.citrus.dsl.testng.TestNGCitrusTestDesigner;
+import com.consol.citrus.http.client.HttpClient;
+import com.consol.citrus.testng.CitrusParameters;
+import com.consol.citrus.validation.json.JsonMappingValidationCallback;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class MalformedRequestTest extends TestNGCitrusTestDesigner {
+
+	@Autowired
+	private HttpClient restTestClient;
+	@Autowired
+	private TestGlobalProperty initGlobalValues;
+	private ObjectMapper objectMapper = new ObjectMapper();
+	private static String admin_token = null;
+
+	@DataProvider(name = "createRequestDataProvider")
+	public Object[][] createRequestDataProvider() {
+		String emptyPayLoad = "{\"request\":{}}";
+		return new Object[][] {
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createUser",
+						"/api/user/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createOrg",
+						"/api/org/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createCourse",
+						"/api/course/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createPage",
+						"/api/page/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createNote",
+						"/api/note/v1/create" } };
+	}
+
+	/**
+	 * Test for create request without content-type header.
+	 *
+	 * @param requestJson
+	 * @param responseJson
+	 * @param testName
```

Comment:
Method is taking four parm , so need to define all four. and add the description of each param.